crossing-repair.lic

```diff
@@ -12,7 +12,7 @@ class CrossingRepair
   def initialize
     settings = setup
 
-    start_script('performance', ['noclean'])
+    start_script('performance', ['noclean']) unless settings.instrument
```

Comment:
Could you elaborate on why this check was added?

Reply:
Added this after I submitted this PR, was intending to do it on the next one.  Sorry for the mistake.